### PR TITLE
Add workflow template metadata

### PR DESCRIPTION
**Pull request by athomas**

### CODE REVIEW

File: workflow-templates/test-package.properties.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Simple Package Testing Workflow (dart-lang)",
+    "description": "Simple Package Testing Workflow using in the dart-lang org",
```
Comment (mit-mit) [inline workflow-templates/test-package.properties.json:3]: s/using in/used in/

File: workflow-templates/test-package.properties.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Simple Package Testing Workflow (dart-lang)",
```
Comment (mit-mit) [inline workflow-templates/test-package.properties.json:2]: We usually don't use title casing, but sentence casing in Dart communication, so 'Simple package testing workflow'

**Review by mit-mit — APPROVED** [review]
Just a few nits

File: workflow-templates/test-package.properties.json (hunk 1 above)
Comment (athomas) [reply workflow-templates/test-package.properties.json]: Done.

File: workflow-templates/test-package.properties.json (hunk 2 above)
Comment (athomas) [reply workflow-templates/test-package.properties.json]: Done.

**Pull request merged**